### ISSUE TITLE
Can't create external alias when a disabled domain exists

### ISSUE THREAD

**Issue by mundschenk-at** [modoboa/modoboa#1391]
# Impacted versions

* Modoboa: 1.10.2

# Steps to reproduce
1. Create a domain **A** and disable it.
2. Create a second domain **B.**
3. Try to create an alias **FOO@B** that points to **BAR@A**.

# Current behavior
Error message: _Local recipient BAR@A not found_
# Expected behavior
Creation of _external_ alias.

A disabled domain should not be considered a local domain. (I use disabled entry to be able to have DMARC reporting for that external domain.)

**Comment by tonioo** [comment]
@mundschenk-at I don't think we should consider a disabled domain as a non local one... Your use case is pretty special, maybe we could find a better way to allow this?

**Comment by mundschenk-at** [comment]
@tonioo Maybe a separate setting to mark a domain as external?

**Comment by tonioo** [comment]
@mundschenk-at Have you tried with a relay domain?

**Comment by tonioo** [comment]
@mundschenk-at ping

**Comment by mundschenk-at** [comment]
Sorry. No, I didn't get around yet. I'll check this out after WordCamp (so next weekend, most likely).

**Comment by mundschenk-at** [comment]
@tonioo Sorry for the long delay. Using a relay domain (with the appropriate transport settings) works indeed.
